src/main/java/com/cbs/java_essential/ruslan/homework/lesson8/task2/Main.java

```diff
@@ -61,13 +61,7 @@ public static void main(String[] args) {
             } catch (IOException e) {
                 System.out.println(e.getMessage());
             }
-            finally {
-                try {
-                    reader.close();
-                } catch (IOException e) {
-                    System.out.println(e.getMessage());
-                }
-            }
+
```

Comment:
создайте метод и вызывайте в мейне в цикле, это плохая практика использовать такое большое количество трай кеч блоков
```suggestion
 // создайте метод и вызывайте в мейне в цикле
     for (int i = 0; i < workers.length; i++) {
        try {
            createWorker(workerNew);
        } catch (IOException e) {
            System.err.println("некорректный ввод!!!\nповторите еще раз");
            log.warn(e);
        }
    }
    //
    private static Worker createWorker(Worker workerNew) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        System.out.println("Введите ФИО: ");
        workerNew.setInitials(reader.readLine());
        System.out.println("Введите должность: ");
        workerNew.setPosition(reader.readLine());
        System.out.println("Введите год приема: ");
        workerNew.setYear(reader.readLine());
        isDateValid(workerNew.getYear());
        reader.close();
        return workerNew;
    }
```